todo_app/templates/index.html

```diff
@@ -10,9 +10,19 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+      <ol class="list-group mb-4">
+        {% for item in items %}
+        <li>{{ item.title }}</li>
+        {% endfor %}
+      </ol>
     </div>
   </div>
+
+
+<div>Add a new todo item: 
+    <form action="/" method="POST"> 
+        <input type="text" name="item"> 
```

Comment:
Minor, but adding a `required` attribute might be nice to just give us some quick browser-side validation so that the user doesn't accidentally submit an empty form